src/boot/paging_32.zig

```diff
@@ -0,0 +1,31 @@
+const zasm = @import("zasm");
+
+extern var p4_table: [4096]u8;
+
+export fn enable_paging() void {
+    // Load P4 to CR3 register. (CPU uses this to access the P4 table).
+    const raw_p4_addr = @intCast(u64, @ptrToInt(&p4_table[0]));
+    var p4_addr = zasm.PhysAddr.initUnchecked(raw_p4_addr);
+    var cr3_value = zasm.Cr3.Contents{
+        .phys_frame = zasm.PhysFrame.fromStartAddressUnchecked(p4_addr),
+        .cr3_flags = zasm.Cr3Flags.fromU64(0),
+    };
+    zasm.Cr3.write(cr3_value);
```

Comment:
Should we move this closer to the enable paging functionality at the end of the function for closer logical distance?

Reply:
Done 5b2cf089ff3ed4ca185cb4666ced132b071aa1c1.